packages/server/lib/cloud/api/index.ts

```diff
@@ -289,6 +290,9 @@ export type ProtocolMetadata = ArtifactMetadata & {
     size: number
     offset: number
   }
+  afterSpecDurations?: AfterSpecDurations & {
```

Comment:
`AfterSpecDurations` is the shape that the protocol capture codebase will return; we're augmenting that with the total duration of `afterSpec`, so the shape is modified here.